README.md

```diff
@@ -65,7 +65,7 @@
 
 ### Key Features <a name="key-features"></a>
 
-- **Display CATEGORIES**
+- **Display Categories**
 - **Add Categories**
 - **Display Transactions**
 - **Add Transactions**
```

Comment:
- [ ] [REQUIRED] Good job so far! I appreciate what you have done above the required thing by previous reviewers but on you forgot to implement to update your `Deployment` link  and `Presentation Video` because in there showing still `Previous design`, please implement my suggestion 🙏🏼 

![image](https://github.com/Samuelwanza/financier/assets/63915024/6e4b56ab-fa77-4e03-b81d-fc2a533047ab)